### DISCUSSION TITLE
Is there any way to set all auto-fixable rules to `warn`?

### DISCUSSION THREAD

**Discussion by alexlalves** [eslint/eslint#15922] [Q&A] [answered]
For example, these two rules can be automatically fixed by eslint:

* `semi`, from https://eslint.org/docs/rules/semi
* `import/order`, from https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md

Sure, I _could_ set each and every rule to warn, but that beats the purpose of plugins and presets. Is there any viable way of achieving this?

**Comment by aladdin-add** [comment, marked as answer]
afaict, there is no straightforward way to get it in eslint core.
